addons/mail/static/src/models/suggested_recipient_info/suggested_recipient_info.js

```diff
@@ -0,0 +1,85 @@
+odoo.define('mail/static/src/models/suggested_recipient_info/suggested_recipient_info.js', function (require) {
+'use strict';
+
+const { registerNewModel } = require('mail/static/src/model/model_core.js');
+const { attr, many2one } = require('mail/static/src/model/model_field.js');
+const mailUtils = require('mail.utils');
+
+function factory(dependencies) {
+    class SuggestedRecipientInfo extends dependencies['mail.model'] {
+        init(...args) {
+            super.init(...args);
+        }
+
+        static convertData(data) {
+            const parsedEmail = data[1] && mailUtils.parseEmail(data[1]);
+            const data2 = {
+                checked: data[0] ? true : false,
+                partner_id: data[0] ? data[0] : undefined,
+                reason: data[2],
+            };
+
+            if (data2.partner_id) {
+                data2.partner = [
+                    ['insert', {
+                        id: data2.partner_id,
+                        name: parsedEmail[0],
+                        email: parsedEmail[1],
+                    }],
+                ];
+            } else {
+                data2.name = '"' + parsedEmail[0] + '" (' + parsedEmail[1] + ')';
```

Comment:
This is from the original template who build it this way : 
"plop@plap.be" (plop@plap.be)

My guess is that there could be case where you have an actual name but no partner, so it would display : 
"Jean Dupond" (jdupon@tintin.be)

Reply:
Display is of format `"<name>" (email)`. The name should not have the display format.

Reply:
I have moved this to the template